auth/authpb/auth.proto

```diff
@@ -15,3 +15,10 @@ message User {
   bytes password = 2;
   int64 tombstone = 3;
 }
+
+// Role is a single entry in the bucket authRoles
+message Role {
+  int64 ID = 1;
```

Comment:
why do we need ID? we want to allow two roles have same name? I feel name should just be the primary key.